bin/actions-toolkit.js

```diff
@@ -2,6 +2,79 @@
 
 const fs = require('fs')
 const path = require('path')
+const { promisify } = require('util')
+const minimist = require('minimist')
+const { prompt } = require('enquirer')
+const { render } = require('mustache')
+const colors = require('./colors.json')
+const icons = require('./feather-icons.json')
+
+const readFile = promisify(fs.readFile)
+const mkdir = promisify(fs.mkdir)
+const readdir = promisify(fs.readdir)
+const writeFile = promisify(fs.writeFile)
+
+const templateDir = path.join(__dirname, 'template')
+
+/** A predicate function to ensure a string is not empty. */
+const isNotEmpty = value => value.length > 0
+
+const getActionMetadata = () => prompt([
+  {
+    type: 'input',
+    name: 'name',
+    message: 'What is the name of your action?',
+    initial: 'Your action name',
+    validate: isNotEmpty
+  },
+  {
+    type: 'input',
+    name: 'description',
+    message: 'What is a short description of your action?',
+    initial: 'A description of your action',
+    validate: isNotEmpty
+  },
+  {
+    type: 'autocomplete',
+    name: 'icon',
+    message: 'Choose an icon for your action. Visit https://feathericons.com for a visual reference.',
+    choices: icons,
```

Comment:
I want the person using this script to be able to select a valid feather icon name from the icons list. They will have the ability to type and filter their selection - check out the [autocomplete prompt](https://github.com/enquirer/enquirer#autocomplete-prompt) for an example of what this interaction would look like.

Reply:
That is _so rad_. Really really great choice!